pkg/image/v1.19.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// NOTE: This is manually replicated from: https://github.com/kubernetes/kubernetes/blob/v1.18.0/test/utils/image/manifest.go#L208-L248
+
+package image
+
+func (r *RegistryList) v1_19() map[int]Config {
+
+	e2eRegistry := r.E2eRegistry
+	dockerLibraryRegistry := r.DockerLibraryRegistry
+	gcRegistry := r.GcRegistry
+	gcAuthenticatedRegistry := r.GcAuthenticatedRegistry
+	googleContainerRegistry := r.GoogleContainerRegistry
+	invalidRegistry := r.InvalidRegistry
+	privateRegistry := r.PrivateRegistry
+	dockerGluster := r.DockerGluster
+	quayIncubator := r.QuayIncubator
+	promoterE2eRegistry := r.PromoterE2eRegistry
+
+	configs := map[int]Config{}
+	configs[Agnhost] = Config{promoterE2eRegistry, "agnhost", "2.12"}
```

Comment:
I think this might still be the 1.18 list of images? It's missing a couple that I can see here: https://github.com/kubernetes/kubernetes/blob/v1.19.0/test/utils/image/manifest.go#L202-L242 such as an iptables image.

Yet another sign that it would be good to automate this 😞

Reply:
Man awesome catch.  Yeah It's possible I pulled from 1.18. 

*sigh* yes, I will push for this to be better soon.